catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/plugins/UpdateParent.java

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.video.stream.mpegts.plugins;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.codice.alliance.video.stream.mpegts.Context;
+import org.codice.alliance.video.stream.mpegts.framework.CatalogUpdateRetry;
+import org.codice.alliance.video.stream.mpegts.rollover.RolloverActionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import ddf.catalog.data.Metacard;
+import ddf.catalog.operation.UpdateRequest;
+import ddf.catalog.operation.impl.UpdateRequestImpl;
+
+/**
+ * Update the parent metacard by calling an {@link UpdateField} and then by calling the
```

Comment:
❓ Should we mention what will get updated in the parent metacard? I couldn't figure it out.